src/widgetresize/resizer.js

```diff
@@ -188,8 +188,16 @@ export default class Resizer {
 		const unit = this._options.unit;
 		const newValue = ( unit === '%' ? this.state.proposedWidthPercents : this.state.proposedWidth ) + this._options.unit;
 
+		this._options.editor.editing.view.change( writer => {
+			writer.setStyle( 'width', this.state.originalWidth + 'px', this._options.viewElement );
+		} );
+
 		this._options.onCommit( newValue );
 
+		const domHandleHost = this._getHandleHost();
+		const domHandleHostRect = new Rect( domHandleHost );
+		this.redraw( domHandleHostRect );
```

Comment:
I wonder why this is needed and what happens there? AFAICS, the width is set there again. However, I assume that `domHandleHostRect` sizes are equal to the `state.originalWidth` and `state.originalHeight`?